core/src/main/java/com/microsoft/applicationinsights/internal/reflect/ClassDataUtils.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * ApplicationInsights-Java
+ * Copyright (c) Microsoft Corporation
+ * All rights reserved.
+ *
+ * MIT License
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this
+ * software and associated documentation files (the ""Software""), to deal in the Software
+ * without restriction, including without limitation the rights to use, copy, modify, merge,
+ * publish, distribute, sublicense, and/or sell copies of the Software, and to permit
+ * persons to whom the Software is furnished to do so, subject to the following conditions:
+ * The above copyright notice and this permission notice shall be included in all copies or
+ * substantial portions of the Software.
+ * THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+ * PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE
+ * FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+ * OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ */
+
+package com.microsoft.applicationinsights.internal.reflect;
+
+/**
+ * Created by gupele on 5/26/2015.
+ */
+public enum ClassDataUtils implements ClassDataVerifier {
+    INSTANCE;
+
+    private ClassDataVerifier verifier = new ClassDataVerifierImpl();
+
+    public void initialize() {
+        verifier = new ClassDataVerifierImpl();
+    }
+
+    @Override
+    public boolean isClassExists(String className) {
```

Comment:
I'm not sure that `isClassExists` (and `isMethodExists` below) is a valid English. I think a better name then would be to replace `is` with `verify` or `check`, i.e. `verifyClassExists` and `verifyMethodExists`. WDYT?